integration/experiment/frequency_sweep/frequency_sweep.py

```diff
@@ -86,7 +86,8 @@ def report_signals():
 
 
 def trace_signals():
-    return []
+    return ['MSR::UNCORE_PERF_STATUS:FREQ@package', 'MSR::UNCORE_RATIO_LIMIT:MAX_RATIO@package',
```

Comment:
I don't think traces are enabled for the frequency sweep run by default.

Reply:
They are not, that's true.  I was hoping to get this in since it's very commonly requested when we start digging into data.  When someone turns traces on, it would be nice to not have to remember to also add the UNCORE fields.